app/plugin_api_test.go

```diff
@@ -684,6 +684,30 @@ func TestPluginAPIGetPlugins(t *testing.T) {
 	assert.Equal(t, pluginManifests, plugins)
 }
 
+func TestPluginAPIUploadPlugin(t *testing.T) {
```

Comment:
can we expand on this to include a test that successfully uploads a plugin?

Reply:
done

Reply:
doesn't look like this was updated?